deployment/deploy.yml

```diff
@@ -53,25 +59,7 @@ objects:
         containers:
         - image: quay.io/cloudservices/platform-changelog-ui:${IMAGE_TAG}
           imagePullPolicy: IfNotPresent
-          livenessProbe:
-            failureThreshold: 10
-            httpGet:
-              path: /
-              port: 8080
-              scheme: HTTP
-            initialDelaySeconds: 30
-            periodSeconds: 5
-            timeoutSeconds: 2
           name: platform-changelog-ui
-          readinessProbe:
-            failureThreshold: 10
-            httpGet:
-              path: /
-              port: 8080
-              scheme: HTTP
-            initialDelaySeconds: 30
-            periodSeconds: 5
-            timeoutSeconds: 2
```

Comment:
Oh, I cut out the liveness and readiness probes because they were giving me fits.  We probably need to add these back in and make sure they work as expected.